spec/features/blacklight_customizations/record_toolbar_spec.rb

```diff
@@ -13,7 +13,7 @@
 
       within "div.navbar-header" do
         expect(page).to_not have_css("button.navbar-toggle", visible: true)
-        expect(page).to_not have_css("a.btn.btn-sul-toolbar", text:"Back to results", visible: true)
+        expect(page).to have_css("a.btn.btn-sul-toolbar", text:"Back to results", visible: true)
```

Comment:
I'm not exactly sure what this scenario (`/spec/features/blacklight_customizations/record_toolbar_spec.rb:9`) is actually testing. This behavior doesn't seem to work in the app right now (see demo), but it's also not failing this particular test.

![toolbar-hide](https://user-images.githubusercontent.com/5402927/29146798-11e85276-7d18-11e7-8b47-b6843bfa425c.gif)